src/schema/matchmaking/cancelled.ts

```diff
@@ -10,7 +10,14 @@ export default defineEndpoint({
         "Server may send this event at any point when the user is queuing to indicate that the user has been booted out the matchmaking system.",
     event: {
         data: Type.Object({
-            reason: UnionEnum(["intentional", "server_error", "party_user_left"]),
+            // * intentional: the player left matchmaking
+            // * server_error: something bad happended and the server couldn't maintain
+            //   state so it booted the player out. Retrying may fix the issue.
+            // * party_user_left: a member of the player's party left matchmaking, thus
+            //   forcing the entire party to withdraw
+            // * ready_timeout: The player failed to accept a match within the given
+            //   time window and is thus removed from the matchmaking system.
```

Comment:
That's not the best place for this documentation, because it's not unexpected to read this source when wanting to learn about protocol. So that fits in the description field, or in the markdown docs for matchmaking.

Reply:
I put that in the matchmaking readme and amended the commit